pipeline.py

```diff
@@ -314,7 +314,7 @@ def pipeline(
         use_secret_as_env(run_mt_bench_task, JUDGE_SECRET, {"api_key": "JUDGE_API_KEY"})
 
         final_eval_task = run_final_eval_op(
-            candidate_model=run_mt_bench_task.outputs["best_model"],
+            candidate_model="/output/model/hf_format/candidate_model",
```

Comment:
oh I can update this back to `run_mt_bench_task.outputs["best_model"]` with the change above, should I?